cloud-sql/sqlserver/mssql/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+
+# Use the official lightweight Node.js 10 image.
+# https://hub.docker.com/_/node
+FROM node:10-slim

```

Comment:
Node 10 is pretty old (EOL already) and although we still support it, our samples should use a newer version (like 14 maybe).

Reply:
Updated to 14.